sdk/identity/azure-identity/test/ut/azure_cli_credential_test.cpp

```diff
@@ -221,6 +221,46 @@ TEST(AzureCliCredential, ExpiresIn)
   EXPECT_LE(token.ExpiresOn, timestampAfter + std::chrono::seconds(30));
 }
 
+TEST(AzureCliCredential, ExpiresOnUnixTimestampInt)
+{
+  constexpr auto Token = "{\"accessToken\":\"ABCDEFGHIJKLMNOPQRSTUVWXYZ\","
+                         "\"expiresOn\":\"2022-08-24 00:43:08.000000\","
+                         "\"expires_on\":1700692424}";
```

Comment:
So, if `expires_on` disagrees with `expiresOn`, the value of `expires_on` wins?

Reply:
Yes. If it is present, we read from `expires_on`, and if we fail, we fall back to `ExpiresOn`.